git.md

```diff
@@ -1,2 +1,139 @@
+* Undoing things with Git ([video](https://youtu.be/dZOfEF19yDk),
+[lesson](https://coderefinery.github.io/git-intro/05-undoing)).
+
+## 3. Commit frequently
+
+Committing frequently keeps your commits small. Small commits are more likely
+to be focused; they are easier to review, and easier to merge because they
+offer fewer opportunities for conflicts.
+
+## 4. The work you share should be complete
+
+Each commit and feature should do what it promises, completely; and reverting
+it should undo what that commit or feature meant to do, also completely. If
+the goal of one commit requires changes in two files, that one commit should
+include changes to both files. Similarly, if the goal of a feature requires
+two kinds of changes, that feature should likely include two commits.
+
+## 5. Explain how to test the effect of the changes you made
+
+Testing your code is important, particularly when sharing it with
+others. Automated tests are the best but not the only way to test the effects
+of the changes you made. However you did it, explain how you ensured that the
+code behaves as you expect, and how someone else may reproduce your results.
+
+## 6. Structure each commit message like an email
+
+Like an email, commit messages have a subject and an optional body. The subject
+should start with a line summarizing your changes (aim for 50 characters or
+less). Use the imperative, present tense -- for example, "Fix typo",
+not "Fixed typo" or "Fixes typo". The message body, if present,
+should follow a blank line; wrap around 70 characters; and explain the
+motivation for the change ("why", not "what" or "how").
+
+See also:
+
+* [Example of a commit message](https://github.com/2DegreesInvesting/resources/issues/74).
+* [Commit best practices](https://r-pkgs.org/git.html#commit-best-practices).
+
+## 7. Agree on a workflow
+
+Choose a workflow based on your team's needs and preferences. However you
+choose to work, just make sure to agree on a common workflow that everyone
+follows:
+
+GitHub workflow:
+
+This workflow is 2DII's default. If a project uses a different
+workflow, document prominently (e.g. in README) what that workflow is
+([example](https://github.com/github/gitignore#contributing-workflow)).
+
+* Uses a central repository. 
+* Uses a single long-lived branch: "main"; it never contains broken code.
+* Developers work on "feature" branches (never on the "main" branch), share
+them via pull requests, and peers review the changes before they become a
+part of the main codebase.
+* A new release is created by tagging a particular commit on the main branch.
+
+Gitflow workflow:
+
+* Like the GitHub workflow but uses two long-lived branches: "main"
+and "develop". The main branch stores the official release history,
+and the develop branch serves as an integration branch for features.
+
+A team may agree on custom conventions beyond the "vanilla" workflows listed
+above.  For example, you may agree to prefix a commit-title with "WIP",
```

Comment:
Thanks for giving me access to the reader's mind. 364c648 removes the reference to the "WIP" prefix to avoid distracting the reader with a thought of whether or not that is even possible. 

(BTW, I use WIP a lot locally but -- as you suggest -- I cleanup before I share that work -- so that's irrelevant to this document which focuses on shared work).